biomaj/bank.py

```diff
@@ -780,19 +810,18 @@ def load_session(self, flow=None, session=None):
                         self.use_last_session = True
 
     def remove_session(self, sid):
-        '''
+        """
         Delete a session from db
 
         :param sid: id of the session
         :type sid: long
         :return: bool
-        '''
+        """
         session_release = None
         _tmpbank = self.banks.find_one({'name': self.name})
         for s in _tmpbank['sessions']:
             if s['id'] == sid:
-                session_release = s['release']
-
+                session_release = self.format_release(s['release'], reverse=True)
```

Comment:
if may impact

if session_release is not None:
            BmajIndex.remove(self.name, session_release)

the format_release does not apply for global release mngt, only in pending cases with your updates. so at removal time of index it will not work. In this method, you should differentiate "pending release" from "release".